src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig

```diff
@@ -40,50 +72,102 @@
               </span>
             </a>
           </div>
-          <h2>
-            {{ "Products"|trans({}, 'Admin.Catalog.Feature') }}
-            {% if has_category_filter == true and activate_drag_and_drop == true %}
-              {{ "(ordering)"|trans({}, 'Admin.Catalog.Feature') }}
-            {% else %}
-              {% if has_filter == true %}
-                {{ "(filtered)"|trans({}, 'Admin.Catalog.Feature') }}
-              {% endif %}
-            {% endif %}
-            {% if has_filter == true %}{{ product_count_filtered }} / {{ product_count }}{% endif %}
-          </h2>
         </div>
       </div>
 
       <div class="row">
-        <div id="product_catalog_category_tree_filter" class="col-md-8">
-          {{ form_widget(categories) }}
+        <div class="col-md-1">
+          <div class="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                id="bulk_action_select_all"
+                onclick="$('#product_catalog_list').find('table td.checkbox-column input:checkbox').prop('checked', $(this).prop('checked')); updateBulkMenu();"
```

Comment:
Would you mind to store this behavior inside this own function ? Maybe ``syncCheckboxes()`` or something ?